test/parallel/test-file-read-noexist.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var path = require('path');
-var fs = require('fs');
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
 
 var filename = path.join(common.fixturesDir, 'does_not_exist.txt');
 fs.readFile(filename, 'latin1', common.mustCall(function(err, content) {
```

Comment:
Nit: Can we add a line after `assert.ok(err)` like `assert.strictEqual(err.code, 'ENOENT')` or whatever the right error code is? I assume it will be the same across platform, but correction welcome.